app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Ways to train if you have a degree"
-heading: "If you have a degree"
+heading: "If you have or are studying for a degree"
```

Comment:
Do we want to change these h1s for people that land on this page and the other ones in this section? And also for accessibility purposes - not great for screen reader users

eg "Training to be a teacher if you have..."

Appreciate this might be a bit of a mouthful though so am certain it could be better than this!

Happy to discuss